app/javascript/packs/wishlist_items.js

```diff
@@ -16,7 +16,15 @@ function onlyUnique(value, index, self) {
     return self.indexOf(value) === index;
 }
 
-document.addEventListener('ajax:error', (event) => {
+function clearErrors() {
+    const container = document.getElementById('errors-list');
+
+    while (container.firstChild) {
+        container.removeChild(container.firstChild);
+    }
+}
```

Comment:
As event listener is properly mounted now, why is this part needed though? Only reason I can think of is for a better clarity, since duplicate errors have already vanished.

Reply:
I also wanted to get rid of it at first, until I realized that without it every form submit will lead to new errors inserted. So unless they are cleared, they will get appended on and on.